packages/console/src/pages/TenantSettings/Subscription/PlanComparisonTable/renderers/MauLimit.tsx

```diff
@@ -0,0 +1,44 @@
+import { type Optional, type Nullable, cond } from '@silverhand/essentials';
```

Comment:
It seems these `FooLimit` components are quite similar. Can we abstract one component to render them all?

Reply:
Updated.